internal/service/cluster/resource_cluster_migration_test.go

```diff
@@ -2,130 +2,141 @@ package cluster_test
 
```

Comment:
Previously the file called `resource_cluster_migration_test.go` had tests related to `StateUpgrader`. That file is now renamed to `resource_cluster_schema_migration_test.go` in this PR.

`resource_cluster_migration_test.go` now contains general migration tests for cluster resource.